actionlist/app/components/contentHeader.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import {
+  StyleSheet,
+  View,
+  Text,
+} from 'react-native';
+import { Colors } from '../lib/commons';
+
+const style = StyleSheet.create({
+  headerText: {
+    fontSize: 10,
+    color: Colors.IUCrimson,
+    fontFamily: 'BentonSansBold, Arial, sans-serif',
+    margin: 10,
+  },
+});
+
+const ContentHeader = ({ title }) => (
+  <View>
+    <Text style={style.headerText}>{title}</Text>
```

Comment:
Yes, the type of `children` should be documented as `React.PropTypes.node`: https://github.com/yannickcr/eslint-plugin-react/issues/7